app/models/validations/property_validations.rb

```diff
@@ -38,6 +38,10 @@ def validate_rsnvac(record)
       record.errors.add :rsnvac, I18n.t("validations.property.rsnvac.referral_invalid")
       record.errors.add :referral, I18n.t("validations.household.referral.rsnvac_non_temp")
     end
+
+    if record.renewal && record.renewal.zero? && record.rsnvac == 14
```

Comment:
```suggestion
    if record.renewal.present? && record.renewal.zero? && record.rsnvac == 14
```

`present?` if a good habit, but not a show stopper here.